t/plugin/openid-connect.t

```diff
@@ -1487,3 +1487,140 @@ GET /t
+            local code, body = t('/apisix/admin/routes/1',
+                 ngx.HTTP_PUT,
+                 [[{ "plugins": {
+                            "openid-connect": {
+                                "client_id": "kbyuFDidLLm280LIwVFiazOqjO3ty8KH",
+                                "client_secret": "60Op4HFM0I8ajz0WdiStAbziZ-VFQttXuxixHHs2R7r7-CW8GR79l-mmLqMhc-Sa",
+                                "discovery": "https://samples.auth0.com/.well-known/openid-configuration",
+                                "redirect_uri": "https://iresty.com",
+                                "ssl_verify": false,
+                                "timeout": 10,
+                                "bearer_only": true,
+                                "scope": "apisix",
+                                "use_jwks": true,
+                                "token_signing_alg_values_expected": "RS256"
+                            }
+                        },
+                        "upstream": {
+                            "nodes": {
+                                "127.0.0.1:1980": 1
+                            },
+                            "type": "roundrobin"
+                        },
+                        "uri": "/hello"
+                }]],
+                [[{ "node": {
+                        "value": {
+                            "plugins": {
+                                "openid-connect": {
+                                    "client_id": "kbyuFDidLLm280LIwVFiazOqjO3ty8KH",
+                                    "client_secret": "60Op4HFM0I8ajz0WdiStAbziZ-VFQttXuxixHHs2R7r7-CW8GR79l-mmLqMhc-Sa",
+                                    "discovery": "https://samples.auth0.com/.well-known/openid-configuration",
+                                    "redirect_uri": "https://iresty.com",
+                                    "ssl_verify": false,
+                                    "timeout": 10,
+                                    "bearer_only": true,
+                                    "scope": "apisix",
+                                    "use_jwks": true,
+                                    "token_signing_alg_values_expected": "RS256"
+                                }
+                            },
+                            "upstream": {
+                                "nodes": {
+                                    "127.0.0.1:1980": 1
+                                },
+                                "type": "roundrobin"
+                            },
+                            "uri": "/hello"
+                        },
+                        "key": "/apisix/routes/1"
+                    },
+                    "action": "set"
+                }]]
+                )
+
+            if code >= 300 then
+                ngx.status = code
+            end
+            ngx.say(body)
+        }
+    }
+--- request
+GET /t
+--- response_body
+passed
+--- no_error_log
+[error]
+
+
+
+=== TEST 26: Access route with valid token.
```

Comment:
This test fails with error "openid-connect.lua:268: phase_func(): OIDC introspection failed: JWT doesn't specify kid but the keystore contains multiple RSA keys"

Reply:
Ahm. yes indeed. I checked the tokens that are already part of the test suite, but sadly none of them have a kid (key id) in its header. I have searched on the auth0 webpage for matching example tokens, but could not find one.

Reply:
@tzssangglass Can you provide some help? thanks.

Reply:
ok, I will.

Reply:
any update？ @tzssangglass

Reply:
will do this in the next two days.

Reply:
@fbartels , we can use `keycloak` instead of `samples.auth0` in the test.

Reply:
@starsz, it will be great if you can show an example for you suggestion

Reply:
OK. @moonming Let me have a try.